ossdbtoolsservice/object_explorer/object_explorer_service.py

```diff
@@ -352,16 +352,18 @@ def _generate_session_uri(params: ConnectionDetails, provider_name: str) -> str:
         if provider_name == utils.constants.PG_PROVIDER_NAME:
             utils.validate.is_not_none_or_whitespace('params.database_name', params.options.get('dbname'))
         utils.validate.is_not_none('params.port', params.options.get('port'))
+        utils.validate.is_not_none_or_whitespace('params.groupId', params.options.get('groupId'))
```

Comment:
Just so I understand, this does not mean it must always have group defined right? Or do we already have a default group

Reply:
ADS has a default “ungrouped” group ID which is used as default if we don’t specify the group!